roles/callhome_configure/tasks/configure.yml

```diff
@@ -156,3 +156,46 @@
       - scale_callhome_params.callhome_schedule is defined 
       - scale_callhome_schedule_status|bool
   run_once: true
+  when: scale_callhome_params.is_enabled | bool
+
+- block:
```

Comment:
@prathamesh0402 Just move complete block into separate file i.e `callhome_unconfigure.yml`

and also please use new flag to unconfigure i.e `scale_callhome_state=absent`